resources/js/Molecules/Card.tsx

```diff
@@ -22,13 +23,15 @@ const Card: React.FC<IProps> = ({ heading, hasPadding }) => {
                                 {heading}
                             </h3>
                         )}
-                        <dd className="text-sm leading-5 font-medium text-gray-500">
+                        {/*<dd className="text-sm leading-5 font-medium text-gray-500">
```

Comment:
We could probably delete this commented out example, now that we have a real example of using the card and passing in children for the card contents.